main.py

```diff
@@ -245,8 +263,8 @@ def resize_app(self) -> None:
         y_coords = int(self.wm_maxsize()[1] / 2 - minimum_height / 2)
 
         # Place app and make the minimum size the actual minimum size (non-infringable)
-        self.geometry(
-            f"{minimum_width}x{minimum_height}+{x_coords}+{y_coords}")
+        self.geometry(f"{minimum_width}x{
```

Comment:
Just put the whole string on its own line instead

Reply:
Well it was due to the Black formatting. I noticed that it does this to long strings.